tests/unit/models/test_ledger_object.py

```diff
@@ -0,0 +1,953 @@
+from unittest import TestCase
```

Comment:
These tests should probably be split into their own files, to mimic the structure of the other models.